actix-web was removed from the default features in
https://github.com/meilisearch/deserr/commit/4657d42d2c6efb839345718182ab193730db5817,
but it's needed for meilisearch.

--- a/meilisearch/Cargo.toml
+++ b/meilisearch/Cargo.toml
@@ -28 +28 @@
-deserr = "0.5.0"
+deserr = { version = "0.5.0", features = ["default", "actix-web"] }
